### PR TITLE
refactor/play_sound_in_ovos_audio

### PR DESCRIPTION
**Pull request by JarbasAl**
companion to https://github.com/OpenVoiceOS/ovos-audio/pull/25

### CODE REVIEW

File: ovos_dinkum_listener/service.py [hunk 1]
```diff
@@ -508,12 +506,7 @@ def _hotword_audio(self, audio_bytes: bytes, ww_context: dict):
 
             if sound:
                 LOG.debug(f"Handling listen sound: {sound}")
-                try:
-                    sound = resolve_resource_file(sound, config=self.config)
-                    if sound:
-                        play_audio(sound)
-                except Exception as e:
-                    LOG.warning(e)
+                self.bus.emit(Message("mycroft.audio.play_sound", {"uri": sound}))
```
Comment (NeonDaniel) [inline ovos_dinkum_listener/service.py:509]: Should this include some message context (routing, ww context?)

Comment (JarbasAl) [reply ovos_dinkum_listener/service.py:509]: this is the source message, just like the utterance message, it is not an answer to anything, but should probably include context for source and destination

ovos-audio side should probably ignore these messages like it does TTS etc for non-native sources

File: ovos_dinkum_listener/service.py [hunk 2]
```diff
@@ -505,8 +505,13 @@ def _hotword_audio(self, audio_bytes: bytes, ww_context: dict):
             event = ww_context.get("event")
 
             if sound:
+                context = {'client_name': 'ovos_dinkum_listener',
+                           'source': 'audio',  # default native audio source
+                           'destination': ["skills"]  # really should be renamed to "core"
+                           }
```
[the remark this answers is not in the document]
Comment (JarbasAl) [reply ovos_dinkum_listener/service.py:511]: there were some notes around this in our docs, for historical reasons the listener is called "audio", we cant easily rename this without messing the general routing, this context is the same as the utterance message

in practice what matters is the native_sources under mycroft.conf, so we can refactor this in a later PR all over the board

[the remark this answers is not in the document]
Comment (JarbasAl) [reply ovos_dinkum_listener/service.py:511]: created an issue in ovos-config for this

**Pull request merged**